vagrant/Makefile

```diff
@@ -41,12 +41,12 @@ client1.test:
 	@$(VSSH) client1 "sudo make -C /root/samba-integration-tests test"
 
 clean_vagrant:
-	@$(VAGRANT) destroy -f
-	@rm -rf .vagrant
+	-$(VAGRANT) destroy -f
+	-rm -rf .vagrant
 
 clean_generated_files:
-	@rm -f Vagrantfile ansible/distro-vars.yml ansible/vagrant_ansible_inventory ansible/ssh-config-setup vagrant_ansible_inventory ssh-config-host
+	-rm -f Vagrantfile ansible/distro-vars.yml ansible/vagrant_ansible_inventory ansible/ssh-config-setup vagrant_ansible_inventory ssh-config-host
```

Comment:
same here

Reply:
https://www.gnu.org/software/make/manual/html_node/Errors.html#Errors

To ignore errors. We can have a partially initialised setup which needs to be cleaned up. ie. say files created by vagrant not started yet. The '-' instructs the Makefile to ignore errors and proceed with subsequent commands.